lib/datadog/appsec/actions_handler/stack_trace/collector.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require_relative 'frame'
+
+module Datadog
+  module AppSec
+    module ActionsHandler
+      module StackTrace
+        # Represent a stack trace with its id and message in message pack
+        module Collector
```

Comment:
what is the responsibility of this `Collector` module? As I see it is only taking a slice from the stack trace depending on some factors, but it's not collecting anything